src/token.rs

```diff
@@ -103,7 +103,7 @@ pub struct GoogleSigninRequiredClaims {
     expires_at: u64,
 
     #[serde(rename = "nbf")]
-    not_before: u64,
+    not_before: Option<u64>,
```

Comment:
Decided to use the old fuchsnj@gmail.com token in test, which didn't have nbf claim.